packages/braid-design-system/src/lib/components/Autosuggest/Autosuggest.docs.tsx

```diff
@@ -482,6 +464,130 @@ const docs: ComponentDocs = {
           </>,
         ),
     },
+    {
+      label: 'Suggestion highlights',
+      description: (
+        <>
+          <Text>
+            A portion of each suggestion item can be highlighted to indicate
+            portions matching the input value. You can use the{' '}
+            <Strong>automaticHighlights</Strong> prop to highlight either the{' '}
+            <Strong>matching</Strong> or <Strong>remaining</Strong> portion of
+            each suggestion.
+          </Text>
+        </>
```

Comment:
Bikeshedding documentation. Feel like this is a bit punchier:

```suggestion
        <Text>
          Suggestion items can be highlighted based on the input value using
          the <Strong>automaticHighlights</Strong> prop. Choose between
          highlighting the <Strong>matching</Strong> or{' '}
          <Strong>remaining</Strong> portion of each suggestion.
        </Text>
```